port_ocean/config/settings.py

```diff
@@ -68,6 +68,8 @@ class IntegrationConfiguration(BaseOceanSettings, extra=Extra.allow):
     initialize_port_resources: bool = True
     scheduled_resync_interval: int | None = None
     client_timeout: int = 60
+    # Determines if Port should generate resources such as blueprints and pages instead of ocean
+    create_resources: bool = False
```

Comment:
Enum: `create_port_resources_origin` : `local` | `port`, where the default is `local`